manifests/cf-manifest/operations.d/510-add-prometheus-blackbox-exporter.yml

```diff
@@ -0,0 +1,27 @@
+---
+# NOTE: This is symlinked to manifests/prometheus/operations.d/...
```

Comment:
I don't think this comment still applies.

Reply:
```suggestion
```

Yep, right you are.